test/data/devicegraphs/output/windows-pc.yml

```diff
@@ -27,6 +27,9 @@
         file_system: swap
         mount_point: swap
 
+    - free:
```

Comment:
Checking for alignment instead of blindly adding that min_grain should save us from these cases in which we introduce a gap. But that's for a second iteration. Before the Christmas break, fixing the main bug and not killing Windows should be enough.

Reply:
That will not be enough; we haven't even started to test for cases where we start with a misaligned setup.